### DISCUSSION TITLE
How to create impressive Readme?

### DISCUSSION THREAD

**Discussion by Nikhil8557** [community/community#108918] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

I'm wondering what is a good profile readme is like

**Comment by Nikhil8557** [comment, marked as answer]
Check https://github.com/Nikhil8557

**Comment by dedsec-a6** [comment]
Make sure to check this one out.
https://rahuldkjain.github.io/gh-profile-readme-generator/

easy and reliable.
just copy and make a readme file with your username and create the repository.
